core/net/ipv6/uip-nd6.h

```diff
@@ -60,11 +60,23 @@
 /** \name RFC 4861 Host constant */
 /** @{ */
 /** \brief Maximum router solicitation delay */
+#ifndef UIP_ND6_MAX_RTR_SOLICITATION_DELAY
```

Comment:
Shouldn't you check for UIP_CONF_ND6_MAX_RTR_SOLICITATION_DELAY? Similarly for UIP_CONF_ND6_RTR_SOLICITATION_INTERVAL below.